SQFormDocs/docs/inclusion_list.md

```diff
@@ -0,0 +1,71 @@
+---
+id: inclusion_list
+title: InclusionList
+---
+
+A field allowing a user to select multiple values to include. <br />
+
+![InclusionList - not selected](../../images/SQFormInclusionListExample1.png) ![InclusionList - selected](../../images/SQFormInclusionListExample2.png) <br />
+
+## How to Use
+
+```js
+import {
+  SQForm,
+  SQFormInclusionList,
+  SQFormStoryWrapper,
+} from '@selectquotelabs/sqform';
+return (
+  <SQFormStoryWrapper
```

Comment:
SQFormStoryWrapper is only being used for our Storybook examples. We don't export that component for use, so they wouldn't be able to follow this example.